binder/postBuild

```diff
@@ -21,4 +21,4 @@
 # Jupyter notebook.
 
 VERSION=$(python -c "exec(open('python/pyspark/version.py').read()); print(__version__)")
-pip install plotly "pyspark[sql,ml,mllib,pandas_on_spark]<=$VERSION"
+pip install plotly "pyspark[sql,ml,mllib,pandas_on_spark]==$VERSION"
```

Comment:
Oh actually it was intentional because the VERSION is like 3.3.0.dev but released PySpark is like 3.2.0

Reply:
I think we should instead do something like force reinstall

Reply:
Oh okay got it now. You mean we should merge this, rebuild and revert this change to retrigger the binder image build?